src/theme/light.js

```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+/*const Colors = {
+  primaryColor: '#5566dd',
+  disabledColor: '#cccccc',
+};*/
```

Comment:
Try not to leave commented code on GitHub. If you are not using it, it is better to delete it.

Reply:
Done, I've already deleted.